wpeverest/everest-forms
# Fix - Format likert field data in seperate column wise while  exporting entry

### All Submissions:


### Changes proposed in this Pull Request:

Previously, The Likert field data is exported in single column, but now the data is exported in separate column.
![image](https://github.com/user-attachments/assets/e9e4d1a4-5c6a-4f32-98a1-981a485f59f2)

### How to test the changes in this Pull Request:

1. Drag a likert field and save same entries.
2. Go to tool > export > select the that form
3. Export entry and verify. also verify by importing that data.

### Types of changes:


### Other information:



### Changelog entry

> Fix - Format likert field data in seperate column wise while  exporting entry.
